prisma/schema.prisma

```diff
@@ -143,6 +143,7 @@ model Proposal {
   supervisedBy      UserProposalSupervision[]
   applications      ProposalApplication[]
   receivedFeedbacks UserProposalFeedback[]
+  AdminInfo         AdminInfo[]
```

Comment:
this is a one to one relation -> one proposal can have only one admin info associated. so no []

Reply:
changed to AdminInfo? (unrequired)